crates/teloxide-core/CHANGELOG.md

```diff
@@ -453,7 +453,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
-- Typo: `ReplyMarkup::{keyboad => keyboard}` ([#69][pr69])
```

Comment:
Can't keep typos even intentionally, so had to reword the changelog entry

Reply:
You made a typo while typing typo 😅 

But also: I think it's necessary to keep the old names in changelog, so I do not like this change...

Reply:
I can add changelog file to ignored files in the config. Though, is this worth doing it? We would lose the checks for typos for the entire file, which consists of narrative plain text that is easy to insert typos into.

I could rewrite this changelog entry smth like "Fix typo in `ReplyMarkup::keyboa^d`" to make `typos` not detect this as an identifier. This changelog entry is soon to be 3 years old. I think people have already migrated from that version of `teloxide` and this update won't ever be read. Therefore, I bet it makes sense to accept this as a bearable limitation. The tradeoff of not checking the changelog seems worse to me, especially because with `typos` the probability that there will be "Fixed a typo" entry in the changelog is approaching zero 💪

Reply:
> I can add changelog file to ignored files in the config

I'm surprised that there isn't a way to ignore a word in a file (only ignoring files or words globally).

---

Does something like ````<code>ReplyMarkup::keyboa<!-- -->d<code>```` work to avoid being flagged by `typos`?

Reply:
Nope, it must be the perfect workaround :smiling_face_with_three_hearts:, I'll try to do that

Reply:
Yep, workef as expected :heavy_check_mark: